plugins/modules/vmware_recommended_datastore.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_recommended_datastore
+short_description: Returns the recommended datastore from a SDRS-enabled datastore cluster
+description:
+- This module provides the recommended datastore name from a datastore cluster only if the SDRS is enabled for the specified datastore cluster
+author:
+- Unknown (@MalfuncEddie)
+- Alina Buzachis (@alinabuzachis)
+- Abhijeet Kasurde (@Akasurde)
+notes:
+- Tested on vSphere 6.7
+requirements:
+- python >= 2.7
+- PyVmomi
+options:
+  datacenter:
+    description:
+    - Name of the datacenter.
+    default: 'ha-datacenter'
```

Comment:
Storage DRS enabled datastore cluster is managed on vCenter, so here the parameter "hostname" is always the vCenter hostname, I think this default value 'ha-datacenter' is not necessary, as it's for standalone ESXi host.